.github/workflows/build-images-action.yml

```diff
@@ -23,6 +23,20 @@ jobs:
       QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
       QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
       SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+  build_ironic_cs10:
+    name: Build Ironic container image based on CentOS Stream 10
+    env:
```

Comment:
Tested this with `gh act` locally.

`Line: 27 Column 5: Failed to match workflow-job: Line: 28 Column 5: Unknown Property env`

Reply:
I see also other errors running act and from the github workflow VS extension, but I'm not sure they are correct, maybe I'm overlooking something
I based my change on https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#defining-environment-variables-for-a-single-workflow

Reply:
ooooook so apparently the env does not work with uses in that way, it seems you need to add env inside  with
it does pass both act and VS code check now